packages/form-layout/theme/lumo/vaadin-form-layout-styles.js

```diff
@@ -6,7 +6,7 @@ registerStyles(
   css`
     :host {
       --vaadin-form-layout-column-spacing: var(--lumo-space-l);
-      --vaadin-form-layout-row-spacing: 0;
+      --vaadin-form-item-row-spacing: 0;
```

Comment:
This is needed to prevent the default `1em` value: https://github.com/vaadin/web-components/pull/8638/files#diff-6ad2d1beab300f671707a82e473d6377e11eb67525696d17cc7ceb3097b6bd3fR54